AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -279,6 +228,80 @@ std::map< int, std::vector<int> > RayTraceCorrelator::SetupPairs(
     return pairs;
 }
 
+std::pair< 
+    std::vector< std::vector< std::vector< int > > >,
+    std::vector< std::vector< std::vector< double > > > > RayTraceCorrelator::GetArrivalDelays(std::map<int, std::vector<int > > pairs){
```

Comment:
sorry I'm having trouble with this git diff. This is still the right data type for this function? It's not a TH2D?

Reply:
Nevermind I think I see what's happening.

Reply:
Ok